mantis-server/mantis-server-agent/src/main/java/io/mantisrx/server/agent/MachineDefinitionUtils.java

```diff
@@ -17,8 +17,17 @@
 
 import io.mantisrx.common.WorkerPorts;
 import io.mantisrx.runtime.MachineDefinition;
+import io.mantisrx.runtime.loader.config.WorkerConfiguration;
+import java.util.Optional;
 
 public class MachineDefinitionUtils {
+    /**
+     * Creates a MachineDefinition object using system parameters.
+     *
+     * @param workerPorts Worker ports object.
+     * @param networkBandwidthInMB Network bandwidth capacity in MB.
+     * @return MachineDefinition object.
+     */
     public static MachineDefinition sys(WorkerPorts workerPorts, double networkBandwidthInMB) {
         return new MachineDefinition(
             Hardware.getNumberCPUCores(),
```

Comment:
Let's maybe get rid of this and just have the user specify the node they are running the TE on.

Reply:
It appears like it's already not being used.